docs/examples.rst

```diff
@@ -293,3 +293,45 @@ prettyprints the single dimension dict:
             "section_pk": 707,
         },
     }
+
+How to Data Upload
+^^^^^^^^^^^^^^^^^^^
+
+Currently the API supports the ability to search for data groups one owns, check the processing status, and upload valid DataFrames. One must still upload data through the UI - specifying the schema. The API is meant to programmatically refresh the data.
```

Comment:
maybe start something with "The API is meant to programmatically refresh existing upload. The initial upload that specifies the schema must still be uploaded via UI."

Reply:
thx